icons/meson.build

```diff
@@ -3,23 +3,22 @@ fs = import('fs')
 foreach flavour: ['default', 'mate']
   if get_option(flavour)
     suffix = flavour == 'default' ? '' : '-@0@'.format(flavour)
+    icon_dir = 'Yaru' + suffix
```

Comment:
@Muqtxdir why did you created this new var?
The `theme_name` one is exactly the same.

Reply:
I added `icon_dir` so that if someone changes `meson.project_name()` from `Yaru` to something else (_for ex: `MyTheme`_), `theme_name` wouldn't be sufficient for that as it would look for `MyTheme` and `MyTheme-mate` icon folders instead of `Yaru` and `Yaru-mate`

Thanks for the review !

Reply:
It's a very nice idea!
I never thought at this problem :)

But maybe doing this in a separated PR?

Reply:
ok, I'll separate them in another PR and revert that change.

Reply:
Thanks :)

Reply:
done, I've reverted the changes made to `icons/meson.build`